src/app/cli/src/init/graphql_queries.ml

```diff
@@ -284,33 +284,37 @@ query user_commands($public_key: PublicKey) {
 }
 |}]
 
-module Pooled_snapp_commands =
+(* Keep this query parsed at compile time to ensure there are no errors + pull the * correct mutation variables. We don't include the full parties selectors because * it's tediuos to keep up-to-date, instead we'll generate those dynamically in an * other query and swap that in when the query is executed (see below). *)
+module Pooled_snapp_commands_partial =
 [%graphql
 {|
 query snapp_commands($public_key: PublicKey) {
   pooledSnappCommands(publicKey: $public_key) {
     id
     hash
-    nonce
-    feePayer { publicKey @bsDecoder(fn: "Decoders.public_key")
-               nonce @bsDecoder(fn: "Decoders.optional_nonce_from_string")
-               balance { total @bsDecoder(fn: "Decoders.balance") }
-               delegate @bsDecoder(fn: "Decoders.optional_account_id")
-               receiptChainHash @bsDecoder(fn: "Decoders.optional_receipt_chain_hash_from_string")
-               snappUri
-               tokenSymbol
-               timing { initialMinimumBalance @bsDecoder(fn: "Decoders.optional_balance")
-                        cliffTime @bsDecoder(fn: "Decoders.optional_global_slot")
-                        vestingPeriod @bsDecoder(fn: "Decoders.optional_global_slot")
-                        vestingIncrement @bsDecoder(fn: "Decoders.optional_amount")
-                      }
-              }
-    fee @bsDecoder(fn: "Decoders.fee")
-    feeToken @bsDecoder(fn: "Decoders.token")
+    failureReason
+    parties { memo }
```

Comment:
We only use this "partial" query to verify that this query shape is valid at compile-time; at runtime we swap it for the "full" query just below (which cannot be verified at compile-time). I think the comment above this query explains this, but let me know if that is insufficient.

Reply:
The concern I had is echoed in one of the later comments; because we generate `parse` from this, surely the returned OCaml type will only have the `memo` accessor?

Reply:
ah good point about `parse`, anyway this is resolved because the query is generated at compile-time now